lib/mapping/loadMapping.js

```diff
@@ -2,38 +2,47 @@ const merge = require('lodash.merge');
 const rcs = require('rcs-core');
 const json = require('json-extra');
 
-const loadMapping = (pathString, options) => {
+function loadReversed(selectors) {
```

Comment:
Also here please change to following:

```js
const loadReversed = selectors => (
	Object.entries(selectors).reduce((prev, [key, value]) => ({
    	...prev,
		[key.charAt(0) + value]: key.slice(1, key.length),
	}), {})
);
```

Reply:
Nope. ava or nyc does not accept object destructuring (I had to use `_.merge` instead).
Not sure it's more understandable this way anyway.
I'm going to do something in between without reduce